src/utils/Store.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { createContext, SetStateAction, useMemo, useState } from 'react';
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+import { Comment } from '../types/Comment';
+import { ErrorTypes } from '../types/ErrorTypes';
+
+interface Children {
+  children: React.ReactNode;
+}
+
+type ActiveUser = User | null;
+
+interface ActiveUserContextType {
+  activeUser: ActiveUser;
+  setActiveUser: React.Dispatch<SetStateAction<ActiveUser>>;
+}
+
+export const ActiveUserContext = createContext<ActiveUserContextType>({
+  activeUser: null,
+  setActiveUser: () => {},
+});
+
+export const ActiveUserProvider: React.FC<Children> = ({ children }) => {
+  const [activeUser, setActiveUser] = useState<ActiveUser>(null);
+
+  const value = useMemo(() => ({ activeUser, setActiveUser }), [activeUser]);
+
+  return (
+    <ActiveUserContext.Provider value={value}>
+      {children}
+    </ActiveUserContext.Provider>
+  );
+};
+
+interface Loader {
+  isLoading: boolean;
+  setIsLoading: React.Dispatch<SetStateAction<boolean>>;
+}
+
+export const LoaderContext = createContext<Loader>({
+  isLoading: false,
+  setIsLoading: () => {},
```

Comment:
The default function for 'setIsLoading' should not be an empty function. Consider providing a more meaningful default or using 'undefined'.